flixel/input/gamepad/buttons/XBox360ButtonID.hx

```diff
@@ -1,11 +1,11 @@
-package flixel.input.gamepad;
+package flixel.input.gamepad.buttons;
 
 import flixel.input.gamepad.FlxGamepad;
 
 /**
  * Button IDs for Xbox 360 controllers
  */
-class XboxButtonID
+class XBox360ButtonID
```

Comment:
I guess that also makes the package name incorrect, maybe that one should be `id` instead of `buttons`?